README.md

```diff
@@ -1 +1,88 @@
-# node-modules-object-lab
\ No newline at end of file
+# Using Object Module
+
+## Objectives
+
+1. Write a simple non-npm module with module.exports = {}
+1. Import your module from another file
+1. Invoke a method from the module
+
+
+## Introduction
+
+We have some customer data for testing our application before we implement the real database or REST API. It's a bad idea to store this data in the code, so your task is to organize the data as a separate file. This way it will be easier to switch from this customer data to some other source later.
+
+In this lab, you'll master how to create, export and import an object module, i.e., a static module.
+
+## Instructions
+
+1. Create a file `module.js` with `module.exports = {}`
+2. Copy customer data object from the `customers.txt` (sample is below) into `module.js` and export the data
```

Comment:
This feels super janky. I know we don't cover `require('something.json')` for another few lessons, but since the emphasis of this lessons is on exporting static data from one module and importing it in another, can we reconfigure it so that the student instead needs to add the appropriate `module.exports` to an existing file of customer data?

We could, for example, change `customers.txt` to `customer.js` and task them with exporting the data (but not telling them exactly how, since this is a lab).